orca/templates/db/Dockerfile

```diff
@@ -1,16 +1,37 @@
-#FROM bitnami/mariadb-galera:10.5
-FROM bitnami/mariadb-galera@sha256:cd63ec64f535e1a7554316429d2597cef94a60a0e877740e21a323c81afd2d4c
+FROM ubuntu:jammy
```

Comment:
Let's move to a hash here as well.

Reply:
Done.
I am also wondering, if we should fix the MySQL release too.

Reply:
Version fixed.